deployment/mainnet/l2addresses.json

```diff
@@ -0,0 +1,17 @@
+{
+  "L2ClaimContract": "0x0000000000000000000000000000000000000000",
```

Comment:
Any reason not to put [0xD7BE2Fd98BfD64c1dfCf6c013fC593eF09219994](https://blockscout.lisk.com/address/0xD7BE2Fd98BfD64c1dfCf6c013fC593eF09219994) here?

Reply:
Thanks for catching this!
Looks like I've discarded changes once L2Claim contract was deployed.
Those files are automatically generated so in normal case they don't need to be modified by hand.